projects/flow.groups-2D-random-walk/src/init.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python3
+"""Initialize the project's data space.
+
+Iterates over all defined state points and initializes
+the associated job workspace directories."""
+import logging
+
+import numpy as np
+import signac
+
+logger = logging.getLogger()
+# Lower level to prevent output
+logger.setLevel(logging.WARN)
+
+# The different standard deviations to use for generating moves in the random
+# walk
+STANDARD_DEVIATIONS = np.linspace(start=0.1, stop=1, num=20)
+NUMBER_REPLICAS = 100
+RUN_STEPS = 5_000
+
+
+def main():
+    project = signac.init_project("2D Gaussian Random Walk")
+    for replica in range(NUMBER_REPLICAS):
+        for std in STANDARD_DEVIATIONS:
+            statepoint = {"mean": 0, "std": std, "replica": replica}
+
+            job = project.open_job(statepoint)
+            job.doc.run_steps = RUN_STEPS
+
+            logger.warn(f"Initializing job with state point: {statepoint}.")
```

Comment:
It took several seconds on my local machine to initialize the data space, and the message printed out is a bit too long. I feel like a short line explaining what this script does is clear enough.